config/locales/views/assignments/en.yml

```diff
@@ -1,9 +1,12 @@
 ---
 en:
   assignments:
+    annotation: Annotation
```

Comment:
You don't need to define a new key for this. Instead, you can use the existing key found in `models/annotations/en.yml`, which is also what the menu label uses. Use the plural form.